src/test/resources/payloads/complex_message_with_styles.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
where is used this file and the one below ? 
I was wondering if we should do a complete test with a complexe payload to test the whole feature, a kind of integration test? We can use this file for it indeed.

Reply:
It's used in this test: https://github.com/symphonyoss/messageml-utils/pull/266/files#diff-c827267e464aa82f6741400d41b3625f6fffb0b767f0c46c4782a497671c3896R368-R379 
yes the idea was exactly that, to test on a more complex example so see how that works